testdata/v1beta1/clustertask/pipelines/s2i-python.yaml

```diff
@@ -18,7 +18,11 @@ spec:
       workspace: source
     params:
     - name: url
-      value: https://github.com/sclorg/django-ex
+      value: https://github.com/openshift-pipelines/release-tests.git
+    - name: git-revision
+      value: master
+    - name: folder
```

Comment:
What is this? Task `git-clone` doesn't have such a parameter. You need to add `testdata/s2i_python` to param `PATH_CONTEXT` of `s2i-python` task

Reply:
@jayesh-garg Please create pipelinerun in your namespace, and fix it if there are any errors